packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -18,6 +18,15 @@
 			<div class="toolbar glass">
 				<tera-toggleable-input :model-value="wf.getName()" @update:model-value="updateWorkflowName" tag="h4" />
 				<div class="button-group">
+					<Button
+						id="add-component-btn"
+						icon="pi pi-expand"
+						label="Reset Zoom"
+						@click="resetZoom"
+						size="small"
+						class="white-space-nowrap"
+					/>
```

Comment:
The button should not be green, as it's not a primary button.
```suggestion
					<Button
						id="add-component-btn"
						icon="pi pi-expand"
						label="Reset Zoom"
						@click="resetZoom"
						size="small"
						class="white-space-nowrap"
						severity="secondary"
					/>
```